website/pages/docs/drivers/podman.mdx

```diff
@@ -31,6 +34,18 @@ Since the Nomad agent, nomad-driver-podman plugin binary, and Podman will
 reside on the same host, skip the ssh aspects of the Podman varlink
 documentation above.
 
+## Capabilities
+
+The `podman` driver implements the following [capabilities](/docs/internals/plugins/task-drivers#capabilities-capabilities-error).
+
+| Feature | Implementation |
+| --- | --- |
+| `nomad alloc signal` | false |
+| `nomad alloc exec` | false |
+| filesystem isolation | none |
```

Comment:
Surprised by this but then I saw: https://github.com/hashicorp/nomad-driver-podman/issues/52

Reply:
Yeah, I unfortunately just noticed it was set as None, I took a quick peak and think all we need to do is change it to Image but it's currently released as 0.1.0 as `None` :/